lib/generators/boring/rspec/install/install_generator.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Boring
+  module Rspec
+    class InstallGenerator < Rails::Generators::Base
+      desc "Adds rspec to the application"
+
+      class_option :version, type: :string, aliases: "-v",
+                             desc: "Tell us the gem version which you want to use for the application"
+
+      def add_rspec_gem
+        say "Adding rspec gem", :green
+        rspec_gem_content = generate_gem_content
+        append_to_file "Gemfile", rspec_gem_content
```

Comment:
The gem content would add another `group :development, :test do` to gem file. Instead can you try to use `insert_into_file` with regex to match `group :development, :test do` and add it gem within existing group?

Reply:
It's great idea. I'll do it when I have time :100: